File: Semester 2 (Sipaling Sibuk Tuh, Masokis!)/Nugas Nugus Nugas Nugus/DDP 2/DDP2 - LAB 00 - ALDEN LUTHFI - 2206028932/java/PangkatFactorial.java
import java.util.Scanner;

public class PangkatFactorial {

    public static void main(String[] args) {
        try (Scanner input = new Scanner(System.in)) {
            
            System.out.print("masukkan bilangan utama (n): ");
            int n = input.nextInt();
            System.out.print("masukkan pemangkatan (m): ");
            int m = input.nextInt();

            int hasilFact = 1, hasilPangkat = 1, tempN = n;

            while (tempN > 0) {
                hasilFact *= tempN--;
            }

            for (int i = 0; i++ < m;) {
                hasilPangkat *= n;
            }

            System.out.printf("n pangkat m = %d\n", hasilPangkat);
            System.out.printf("n factorial = %d\n", hasilFact);
        }
    }
}

File: Semester 2 (Sipaling Sibuk Tuh, Masokis!)/Tugas/DDP2/DDP2 - LAB 00 - ALDEN LUTHFI - 2206028932/python/pangkat_factorial.py
from turtle import st


n = int(input("masukkan bilangan utama (n): "))
m = int(input("masukkan pemangkatan (m): "))

# Factorial
hasil_fact = 1
temp_n = n
while temp_n >= 1:
    hasil_fact = hasil_fact * temp_n
    temp_n = temp_n - 1

# Pangkat
hasil_pangkat = 1
for i in range(m):
    hasil_pangkat = hasil_pangkat * n

print("n factorial = " + str(hasil_fact))
print("n pangkat m = " + str(hasil_pangkat))
